_includes/tutorials/ksql-concatenation-columns/ksql/markup/dev/transient-reporting.adoc

```diff
@@ -0,0 +1,35 @@
+Now we write a query where we select the desired fields but we don't have them listed in a comma separated list.  Instead we use the `+` operator between the fields causting them to get concatenated together.
+
+[source, sql]
+----
+
+SELECT FIRST_NAME + ' ' + LAST_NAME +
+       ' purchased ' +
+       CAST(NUM_SHARES AS VARCHAR) + <1>
+       ' shares of ' +
+       SYMBOL AS SUMMARY
+FROM ACTIVITY_STREAM
+EMIT CHANGES
+LIMIT 4;
+
+
+----
+
+<1> The NUM_SHARES field is an `INT` so we need to cast it to a `VARCHAR` as `concatenate` only works with `STRING` types
```

Comment:
What happens if you don't cast a field to a varchar?

Reply:
results in an error, I've updated the text to state as much